services/api/src/schema.js

```diff
@@ -67,6 +67,7 @@ const typeDefs = `
     active_systems_remove: String
     branches: String
     production_environment: String
+    auto_idle: String
```

Comment:
`auto_idle` is defined as a `String` but inside the DB it's actually an `Int`, let's change that here.

Reply:
I've changed this to `Int`.